src/components/TimeControl/util.ts

```diff
@@ -601,6 +601,30 @@ export function durationString(seconds): string { /* {{{ */
 
     return ret.trim();
 } /* }}} */
+export function daysOnlyDurationString(seconds): string { /* {{{ */
+    let days = Math.floor(seconds / 86400);
+
+    function plurality(num, single, plural): string {
```

Comment:
Remove this declaration, use `ngettext` or `npgettext` instead

Reply:
thanks a lot! I hadn't looked into how they work, that cleaned up a lot of LOC 😄